Dizzy/UI Components/DizzyLoadingView.swift

```diff
@@ -40,19 +40,20 @@ class DizzyLoadingView: UIView {
     }
     
     func startLoadingAnimation() {
-        changeLogoVisability()
-        Timer.scheduledTimer(timeInterval: 1.5, target: self, selector: #selector(changeLogoVisability), userInfo: nil, repeats: true)
+        makeLogoSpin()
+        Timer.scheduledTimer(timeInterval: 0.7, target: self, selector: #selector(makeLogoSpin), userInfo: nil, repeats: true)
     }
     
     func stopLoadingAnimation() {
         Timer.cancelPreviousPerformRequests(withTarget: stopLoadingAnimation)
     }
     
-    @objc func changeLogoVisability() {
-        UIView.animate(withDuration: 2.0) {
-            self.loadingIcon.alpha = 1 - self.loadingIcon.alpha
-            self.layoutIfNeeded()
-        }
+    @objc func makeLogoSpin() {
```

Comment:
this code is not good enough.  please return it to the old way or wait until we will meet an i will explain why is it not good enough.

Reply:
I reverted back to original version.
Please explain